server/fileupdater.mjs

```diff
@@ -434,3 +435,12 @@ function v12HandDropShadow(properties) {
     properties.dropShadow = true;
   }
 }
+
+function v13fixedParentDefaults(properties) {
+  if (properties.type == 'card' || properties.type == 'pile' || properties.type == null){
+    if (properties.fixedParent=true)
```

Comment:
In javascript a single `=` is assignment not comparison.

Reply:
Yep.